php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -245,9 +245,11 @@ class NDB_BVL_Battery
     {
         // assert that a battery has already been selected
         $error = $this->_assertBatterySelected();
+        '@phan-var bool $error';
```

Comment:
Why are you doing this? If the error is "unused variable" assigning the variable a type won't fix it. You need to just not assign the function call to anything like

```
$this->_assertBatterySelected()
```

instead of

```
$error = $this->_assertBatterySelected();
```